gnucash/gschemas/org.gnucash.warnings.gschema.xml.in

```diff
@@ -99,6 +99,11 @@
       <summary>Commit changes to a transaction</summary>
       <description>This dialog is presented when you attempt to move out of a modified transaction. The changed data must be either saved or discarded.</description>
     </key>
+    <key name="rec-after-match" type="i">
+      <default>0</default>
+      <summary>Reconcile after matching transactions</summary>
+      <description>This dialog is presented when you are done matching transactions and there is a balance information in your statement.</description>
```

Comment:
I think sentence needs changing, drop the 'a' so 'and there is balance'
Also below.